technische-universitat-dresden-medizin.csl

```diff
@@ -152,7 +152,7 @@
       <date-part name="day" suffix="."/>
     </date>
   </macro>
-  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
+  <citation name-form="short" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" initialize="false" sort-separator="" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true">
```

Comment:
no need to repeat all the attributes you already have in author-short; i.e. delete
``` name-form="short" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"``` and ```initialize="false" sort-separator=""```